hyperstart.go

```diff
@@ -323,7 +323,9 @@ func (h *hyper) bindUnmountContainerRootfs(podID, cID string) error {
 func (h *hyper) bindUnmountAllRootfs(pod Pod) {
 	for _, c := range pod.containers {
 		h.bindUnmountContainerMounts(c.mounts)
-		h.bindUnmountContainerRootfs(pod.id, c.id)
+		if c.state.Fstype == "" {
+			h.bindUnmountContainerRootfs(pod.id, c.id)
```

Comment:
Is there an issue to track this? Not checking a return value for an operation like this sounds dangerous to me. I'd add a reference to an issue in a comment for this line.